archivy/config.py

```diff
@@ -1,3 +1,24 @@
+#!/usr/bin/env python
```

Comment:
What's the point of adding the shebang here? The point of the packaging is that we don't execute the files directly?

Reply:
Small story from this excerpt:

> **sysadmin**: you might have been confused by the unannounced reboot on the system
> **people**: yes, why was the reboot scheduled
> **sysadmin**: Actually, I wanted to source by `source ~/.bashrc` again, but on mistake, I tab completed it to ...  `source ~/.bash_history`
> **people**: silence
> **sysadmin**: Thankfully, I had some day `reboot` in my history, and that got executed. Hopefully, there was no `rm -rf` before the reboot :sweat: 

I can remove the shebangs if you prefer, but I prefer to keep it.

Reply:
I'd appreciate if you could remove the shebangs as they are for stand alone executables and not really appropriate in this type of package.

Reply:
Sure :)